src/nvim/testdir/test_normal.vim

```diff
@@ -1226,29 +1226,6 @@ func Test_normal23_K()
   set iskeyword-=%
   set iskeyword-=\|
 
-  " Only expect "man" to work on Unix
-  if !has("unix")
```

Comment:
Avoid deleting lines in the oldtests. It makes merging upstream changes much easier if the lines exist, but are either commented-out (for 1-2 lines) or skipped.

To avoid indenting the skipped lines (again to avoid merge-conflicts from upstream), we can do an early-return here:

```
if !has("unix") || has('nvim')  " Nvim K uses :terminal. #15398
```